khi_robot_control/include/khi_robot_hardware_interface.h

```diff
@@ -56,8 +56,8 @@ class KhiRobotHardwareInterface : public hardware_interface::RobotHW
     bool hold();
     void deactivate();
     void close();
-    void read( const ros::Time time, const ros::Duration period );
-    void write( const ros::Time time, const ros::Duration period );
+    void read( const ros::Time& time, const ros::Duration& period );
+    void write( const ros::Time& time, const ros::Duration& period );
```

Comment:
Suggestion: add `override` keyword here.

Reply:
Yes, it is better to add.

Reply:
@d-nakamichi: would you want me to add it in this PR?

Reply:
Yes I want, because adding `override` keyword is closely related to this PR.

Reply:
Done in bb6dc4d.